docs/execute-notebooks.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Execute Notebooks with Jupyter
+title: Execute Notebooks During Your Build
 description: MyST can execute notebooks using Jupyter Server, making it possible to build rich websites and documents from text-based notebooks.
-short_title: Executing Notebooks
+short_title: Execute Notebooks During Your Build
```

Comment:
@rowanc1 this should probably be shorter as it's part of the sidebar nav?

Reply:
Yes, probably. :)